deploy.sh

```
@@ -1,6 +1,25 @@
 #!/bin/bash
 echo -e "배포 스크립트 시작\n\n"
 
+# .env 파일 로드
+echo "환경 변수 로드 중..."
+if [ -f ".env" ]; then
+    export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)